packages/data/src/world/mm/great_bay_temple.yml

```diff
@@ -105,7 +104,7 @@
     "Great Bay Temple Central Room": "underwater_walking || hookshot_anywhere"
     "Great Bay Temple Ice Arrow Room": "small_keys_gb(1)"
   events:
-    GB_PIPE_RED: "(soul_octorok && can_use_ice_arrows) || short_hook_anywhere"
+    GB_PIPE_RED: "(soul_octorok && can_use_ice_arrows) || short_hook_anywhere || (has_hover_boots && has_mask_bunny && has_weapon && trick(MM_GBT_RED1_HOVERS))"
```

Comment:
Is this running from the red pipe to the platform with the switch? Does it need a jump slash and bunny or can it be done with a backwalk + backflip?

Reply:
This is running from Wart's room. the pipe is too low.